content/en/data_security/agent.md

```diff
@@ -88,6 +88,100 @@ If you have a requirement to avoid storing secrets in plaintext in the Agent's c
 
 For more information, see the [Secrets Management][18] documentation.
 
+## Telemetry collection
+
+{{< site-region region="gov" >}}
+
+Agent on non-government sites collects environmental, performance, and feature usage information about the Datadog Agent. When the Agent detects a government site, or [ Datadog Agent FIPS Proxy][1] is used, the Agent automatically disables this telemetry collection. When such detection is impossible (such as, for example, if a proxy is being used) Agent telemetry will be emitted but immediately dropped at Datadog's intake. To avoid this data from being emitted in the first place we recommend disabling Agent telemetry explicitly by updating the `agent_telemetry` setting in the Agent configuration file, as shown in the example below.
+
+{{< tabs >}}
+{{% tab "datadog.yaml" %}}
+
+```yaml
+agent_telemetry:
+  enabled: false
+```
+{{% /tab %}}
+{{% tab "Environment variables" %}}
+
+```bash
+DD_AGENT_TELEMETRY_ENABLED=false
+```
+{{% /tab %}}
+{{< /tabs >}}
+[1]: https://docs.datadoghq.com/agent/configuration/agent-fips-proxy/?tab=hostorvm&site=gov
+{{< /site-region >}}
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+Datadog may collect environmental, performance, and feature usage information about the Datadog Agent. This may include diagnostic logs and crash dumps of the Datadog Agent with obfuscated stack traces to support and further improve the Datadog Agent.
+
+You can disable this telemetry collection by updating the `agent_telemetry` setting in the Agent configuration file, as shown in the example below.
+{{< tabs >}}
+{{% tab "datadog.yaml" %}}
+
+```yaml
+agent_telemetry:
+  enabled: false
+```
+{{% /tab %}}
+{{% tab "Environment variables" %}}
+
+```bash
+DD_AGENT_TELEMETRY_ENABLED=false
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+**Telemetry content:**
+| Metadata ([source][1]) |
+| ---------------------- |
+| Machine id             |
+| Machine name           |
+| OS                     |
+| OS version             |
+| Agent version          |
+
+| Metrics ([source][2])                       | Description                                                                                       |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| checks.execution_time                       | Check's execution time in milliseconds                                                            |
+| pymem.inuse                                 | Number of bytes allocated by the python interpreter                                               |
+| pymem.alloc                                 | Total number of bytes allocated by the python interpreter since the start of the agent            |
+| api_server.request_duration_seconds         | Total number of bytes allocated by the python interpreter since the start of the agent            |
```

Comment:
```suggestion
| api_server.request_duration_seconds         | Total number of bytes allocated by the Python interpreter since the start of the Agent            |
```
Is this correct? It's the same as the field above

Reply:
@hestonhoffman, missed your question actually ... looking

Reply:
Do not know how I missed that, fixing